.mk/k8s.mk

```diff
@@ -93,11 +56,12 @@ SPIRE_ENABLED=false
 endif
 
 kubectl = kubectl -n ${NSM_NAMESPACE}
+images_tar = $(subst .tar,,$(filter %.tar, $(shell ls ./scripts/vagrant/images)))
```

Comment:
Not seems right to me to depend on images folder. I suppose since we have helm charts rule our deployments, to use helm charts to list a valid deployment targets?

Reply:
This list is being used only for k8-load-images

Reply:
It is still not god, since if there is no some of images, it will not fail but upload only images in folder.